.travis.yml

```diff
@@ -29,11 +35,24 @@ script:
   # Build maestro
   - ./build.sh
 
+  # Run devicedb in docker
+  - |
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      docker run --rm -d -p 127.0.0.1:9090:9090/tcp senramakri/test_images:devicedb_nocluster
+      sleep 10
+    fi
+
   # Run unit tests
   - cd "../../${upstream}"
   - DEBUG=1 DEBUG2=1 ./build.sh
   - ( cd sysstats && LD_LIBRARY_PATH=../../maestro/vendor/github.com/armPelionEdge/greasego/deps/lib/ go test -v )
   - ( cd maestroConfig && LD_LIBRARY_PATH=../../maestro/vendor/github.com/armPelionEdge/greasego/deps/lib/ go test -v )
+  - ( cd vendor/github.com/armPelionEdge/maestroSpecs && go test -v )
```

Comment:
Double checking. Are we still not able to reliably enable all tests for maestro?

Reply:
Events tests still having some issues. That needs to be looked, but everything else has been added including the new ones. We cannot enable all Network tests as some of them need manual intervention and cannot run in Travis env.

Reply:
@SenRamakri It might be worth tracking those tests in an issue for further work.

Reply:
Note for future Cruz: Internal Jira ticket reopened.